nodeJS/introduction_to_nodeJS/project_basic_informational_site.md

```diff
@@ -1,27 +1,26 @@
-### A Node website!
+### A Node website
 
-By now you should know enough to be dangerous. There's *definitely* more to learn but you already know enough about Node to actually make something! So let's do it! You will be creating a very basic informational site that contains 4 pages: index, about, contact-me and 404. Keep in mind that the content of these pages isn't really all that important. So there's no need to spend a lot of time filling them up or trying to make them look pretty.
+By now you should know enough to be dangerous. There's _definitely_ more to learn but you already know enough about Node to actually make something! So let's do it! You will be creating a very basic informational site that contains 4 pages: index, about, contact-me and 404. Keep in mind that the content of these pages isn't really all that important. So there's no need to spend a lot of time filling them up or trying to make them look pretty.
 
-If you get stuck at anytime, feel free to go back to the [Getting Started lesson](https://www.theodinproject.com/lessons/nodejs-getting-started), especially the W3 schools tutorial at the end.
+If you get stuck at anytime, feel free to go back to the [Getting Started lesson](https://www.theodinproject.com/lessons/nodejs-getting-started).
 
 ### Assignment
 
 <div class="lesson-content__panel" markdown="1">
 
 1. Make a project directory and create the following files inside that directory:
-    - index.html
-    - about.html
-    - contact-me.html
-    - 404.html
-
-2. Create your node.js server file `index.js` and add the code needed to serve the right page according to the url.
-    - [localhost:8080](http://localhost:8080) should take users to index.html
-    - [localhost:8080/about](http://localhost:8080/about) should take users to about.html
-    - [localhost:8080/contact-me](http://localhost:8080/contact-me) should take users to contact-me.html
-    - 404.html should display any time the user tries to go to a page not listed above.
+
+   - index.html
+   - about.html
+   - contact-me.html
+   - 404.html
+
+2. Create your node.js server file `index.js` and add the code needed to serve the right page according to the url. - [localhost:8080](http://localhost:8080) should take users to index.html - [localhost:8080/about](http://localhost:8080/about) should take users to about.html - [localhost:8080/contact-me](http://localhost:8080/contact-me) should take users to contact-me.html - 404.html should display any time the user tries to go to a page not listed above.
```

Comment:
Nit: Each of the bullet points need to be on a new line and nested under the numbered item to render correctly on the site. Same as the 1st assignment if you need an example to follow 😄